mflowgen/common/custom-power-leaf/outputs/power-strategy-dualmesh.tcl

```diff
@@ -0,0 +1 @@
+../../power-scripts/leaf-power-strategy-dualmesh.tcl
```

Comment:
Why is this a symlink? It is only used here, right?

Reply:
Fair point... I had organized it such that I had scripts in one folder, but nodes were kept separate (to share globalnetconnect.tcl) between all of them, but perhaps I should only do that for the script that's actually shared.